tcl/html.tcl

```diff
@@ -869,3 +869,8 @@ proc qc::safe_attributes {} {
 
     return $safe_attributes
 }
+
+proc qc::global_response2html {} {
```

Comment:
Considered that but didn't know if it was a little bit too generic like `data2html`.